pkg/chunk/grpc/grpc.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+import "google/protobuf/empty.proto";
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(BatchWrites) returns (google.protobuf.Empty);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
```

Comment:
Should we call it `QueryIndex(QueryIndexRequest) returns (stream QueryIndexResponse)` for clarity/consistency?